empire

```diff
@@ -990,7 +990,7 @@ def start_restful_api(empireMenu, suppress=False, username=None, password=None,
         for agentNameID in agentNameIDs:
             (agentName, agentSessionID) = agentNameID
 
-            execute_db_query(conn, 'UPDATE agents SET results=? WHERE session_id=?', ['', agentSessionID])
+            execute_db_query(conn, 'DELETE FROM results WHERE agent=? OR agent=?', [agentSessionID, agentName])
```

Comment:
I'm not sure this is what was intended by the original team and I don't think we want to be able to delete individual results, since it is used for reporting.

Agents have a field on them called "results" which is a json string. As far as I can tell it is [used by the CLI as a sort of cache](https://github.com/BC-SECURITY/Empire/blob/master/lib/common/agents.py#L617) for events that gets cleared once they are displayed, and this endpoint does something similar.

IMO, your update of using session_id OR agent_name is needed, but I think it should still just be setting the column in agents to ''. I'm happy to discuss further if you see it different. it does seem like a useless endpoint to me and I may be misinterpreting it.

Reply:
Gotcha, yeah that makes sense from a reporting standpont. I'll revert that.